docs/Testing.md

```diff
@@ -325,6 +325,11 @@ code for the target that is not the build machine:
 
 * ``%target-swiftc_driver``: run ``swiftc`` for the target.
 
+* ``%target-swiftc_driver-emit-module-interface(`` *swift interface path* ``)``
```

Comment:
nit: `swiftc` already presumes the driver, would be more succinct to omit the `_driver` part.

Reply:
That makes sense, I can change this

Reply:
Updated